bin/growpart

```diff
@@ -375,6 +385,14 @@ resize_sfdisk() {
 				"Additional space will go unused."
 	fi
 
+	if [ -n "${free_percent}" ]; then
+		# Shrink max_end to keep X% of whole disk unused
+		# (for overprovisioning)
+		free_percent_sectors=$((sector_num/100*free_percent))
+		debug 1 "reserving ${free_percent_sectors} sectors (${free_percent}%) for overprovisioning"
```

Comment:
minor note in docs/logs I think over-provisioning is hyphenated

Reply:
I've seen both forms of spelling. It might be a British English vs American English difference.

Reply:
As a non-native speaker I'll abstain here, let's just keep it consistent :) FWIW in the Debian codebase "overprovisioning" is way more common:

 - https://codesearch.debian.net/search?q=overprovisioning&literal=1
 - https://codesearch.debian.net/search?q=over-provisioning&literal=1

Reply:
+1 I'm good with overprovisioning. Sorry for delay there on a nit.